### DISCUSSION TITLE
hardware acceleration?

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [OpenSurv/OpenSurv#28] [General]
Does OpenSurv use any hardware acceleration? I didn't see any options in the ```monitor1.yml``` file.

**Comment by SvenVD** [comment]
Depending on the hardware you are running on, mpv will automatically use hardware acceleration, you can finetune mpv behavior with freeform_advanced_mpv_options https://github.com/OpenSurv/OpenSurv/blob/e904ccc128fe56990242b801af3437302a5d79d1/surveillance/etc/monitor1.yml#L96
